frontend/src/components/lease/Details.tsx

```diff
@@ -0,0 +1,34 @@
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router';
+import { Lease } from '../../lib/k8s/lease';
+import { DateLabel, DetailsGrid } from '../common';
+
+export function LeaseDetails() {
+  const { namespace, name } = useParams<{ namespace: string; name: string }>();
+  const { t } = useTranslation('glossary');
```

Comment:
This should be in the `resource` namespace, not glossary, I think. glossary is normally for the type of things and other technical jargon.

Reply:
updated